docs/architecture/concentrated-liquidity-adr.md

```diff
@@ -328,7 +328,7 @@ Let's begin by considering the execution flow of the pool creation message.
 //
 // After the initial liquidity is sent to the pool's account, shares are minted
 // and sent to the pool creator. The shares are created using a denomination in
-// the form of gamm/pool/{poolID}. In addition, the x/bank metadata is updated
+// the form of < swap module name >/pool/{poolID}. In addition, the x/bank metadata is updated
```

Comment:
I actually meant, its possible shares will need to contain min and max ticks which the user is providing liquidity to for CL right?

Reply:
it is possible if necessary

Reply:
Good point. I think we can update this down the road. 

Afaiu the shares are non-fungible between the positions. So we will need to think more about how we want to identify them.

Also, `concentrated-liquidity` module is not hooked up to this yet anyways.